### ISSUE TITLE
GetProhibited exception on create lockdown with create_using_usbmux at iPhone 8 via WiFi

### ISSUE THREAD

**Issue by alex-opeykin** [doronz88/pymobiledevice3#932]
**Test environment**
- Windows 10 Pro 22H2
- iPhone 8, iOS 16.7.7 

**Describe the bug**
GetProhibited exception on create lockdown with create_using_usbmux at iPhone 8 via WiFi
Lockdown created successful when USB connected, but over WiFi failed

**To Reproduce**
1. Connect iPhone to PC with USB
2. Unplug cable
3. Check device connected with `pymobiledevice3 usbmux list`, "ConnectionType": "Network"
4. Try create lockdown connection in py script
`lockdown = create_using_usbmux(identifier="myiphone8idid")`

**Expected behavior**
lockdown created without errors

**Logs**
```
Traceback (most recent call last):
  File "C:\SomeFolder\ios_test_runner.py", line 114, in <module>
    lockdown = create_using_usbmux(identifier="myiphone8idid")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 729, in create_using_usbmux
    return cls.create(
           ^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 138, in create
    lockdown_client._handle_autopair(autopair, pair_timeout, private_key=private_key)
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 532, in _handle_autopair
    self.pair(timeout=timeout, private_key=private_key)
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 60, in _inner_reconnect_on_remote_close
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 331, in pair
    self.device_public_key = self.get_value('', 'DevicePublicKey')
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 60, in _inner_reconnect_on_remote_close
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 439, in get_value
    res = self._request('GetValue', options)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\pymobiledevice3\lockdown.py", line 565, in _request
    raise exception_errors.get(error, LockdownError)(error)
pymobiledevice3.exceptions.GetProhibitedError: GetProhibited
```

**Additional context**
Lockdown created successful when USB connected, but over WiFi failed
Also i can use cli commands, i.e. i cat launch apps with `pymobiledevice3 developer dvt launch `
Also it's work via WiFi on iPhone 14 Pro 16.7.2

**Comment by doronz88** [comment]
When running:

```shell
pymobiledevice3 usbmux list
```

Does the device shows its UDID as its identifier or its mac address?

**Comment by alex-opeykin** [comment]
It's 40 symbols identifiers

**Comment by doronz88** [comment]
I don't understand your answer. Can you provide the exact output?

**Comment by alex-opeykin** [comment]
Connected with USB cable
```
PS C:\Windows\system32> pymobiledevice3 usbmux list
[
    {
        "BuildVersion": "20H330",
        "ConnectionType": "Network",
        "DeviceClass": "iPhone",
        "DeviceName": "iPhone (Alex)",
        "Identifier": "5cc1f8a1cd615887e99ea23e2adb624781b10d58",
        "ProductType": "iPhone10,4",
        "ProductVersion": "16.7.7"
    },
    {
        "BuildVersion": "20H330",
        "ConnectionType": "USB",
        "DeviceClass": "iPhone",
        "DeviceName": "iPhone (Alex)",
        "Identifier": "5cc1f8a1cd615887e99ea23e2adb624781b10d58",
        "ProductType": "iPhone10,4",
        "ProductVersion": "16.7.7"
    }
]
```
when cable unpluged
```
PS C:\Windows\system32> pymobiledevice3 usbmux list
[
    {
        "BuildVersion": "20H330",
        "ConnectionType": "Network",
        "DeviceClass": "iPhone",
        "DeviceName": "iPhone (Alex)",
        "Identifier": "5cc1f8a1cd615887e99ea23e2adb624781b10d58",
        "ProductType": "iPhone10,4",
        "ProductVersion": "16.7.7"
    }
]
```

**Comment by alex-opeykin** [comment]
Hmm, after reboot, this error not occurs on iPhone 8, iOS 16.7.7. Maybe update and reboot helped
But now i get same error on iPhone 11, iOS 16.6. I'll try to update

**Comment by alex-opeykin** [comment]
oh, my bad..
error occurs cauze i forgot to mount developer image on device
Now i get only error ConnectionAbortedError #933
